acceptancetests/assess_network_health.py

```diff
@@ -325,6 +530,7 @@ def ensure_exposed(self, client, series):
             self.setup_expose_test(client, series, exposed)
 
         service_results = {}
+        log.info('MACHINE:\n {}'.format(json.dumps(yaml.safe_load(client.get_juju_output('list-machines', '--format=yaml'))['machines'], indent=4, sort_keys=True))) # VWH
```

Comment:
This needs some linebreaks :-)

Reply:
That line was meant for me debugging my code. The line itself is too long and the command gives excessive output. I'll break it up anyway. :)